processflow.md

```diff
@@ -1,24 +1,30 @@
 2. The [Management API](glossary.md#management-api) writes a batch notification message to the associated notification topic.
 3. The [Data Consumer](glossary.md#data-consumer) receives the batch notification message.
 4. Data Integrator writes the data to the correct Kafka `*.in` topic.
-5. The Consumer _may now_ begin reading the data from the Kafka topic but can choose to wait until step 8 to begin reading the data.
-6. The Data Integrator completes writing all data contained in this batch and it then signals to the Management API that it completed sending the data for the batch.
+5. The Data Consumer **can now** start reading the data from the Kafka topic but can choose to wait until step 8 to begin reading the data.
+6. The Data Integrator completes writing all data contained in this batch. Then, it signals to the Management API that it completed sending the data for the batch.
 7. The Management API writes a batch notification message to the associated notification topic.
-8. The Consumer receives the batch notification message.
+8. The Data Consumer receives the batch notification message.
+
+## Alternate flows
+### Batch termination
+If the Data Integrator encounters an error after creating a batch in step 2 earlier in this topic, the integrator can send a request to the Management API to **terminate** the batch. Then, the Management API writes a batch notification message  to the associated notification topic, and the Data Consumer receives it.
+
+### Interleaved batches
+Health Record Ingestion does not prevent the Data Integrator from writing multiple batches into the same topic at the same time. Each record has a header value that specifies the ["batchId"](glossary.md#batch-id), which is returned from the Management API, so the Data Consumer can distinguish each one. 
 
-## Alternate Flows
-### Batch Termination
-If the Data Integrator encounters an error after creating a batch in step 2, they may send a request to the Management API to _terminate_ the batch. The Management API will then write a batch notification message to the associated notification topic, and the Consumer will receive it.
+To review a batchId in the Management API on GitHub, click [here](https://github.com/Alvearie/hri-api-spec/tree/master/management-api/management.yml#L36). 
 
-### Interleaved Batches
-The HRI does not prevent the Data Integrator from writing multiple batches into the same topic at the same time. Every record will have a header value that specifies the ["batchId"](glossary.md#batch-id), which is returned from the Management API (see [api-spec/management-api/management.yml](https://github.com/Alvearie/hri-api-spec/tree/master/management-api/management.yml#L36)), so the Consumer can distinguish each one. 
+In practice, the Data Integrator can only write one batch at a time. As needed, additional input topics can be created to prevent the interleaving of batches or data types. 
```

Comment:
'the Data Integrator **can** only write' is not correct.  They can write multiple batches at the same time. The original wording is trying to highlight the fact that most of the time, Data Integrators will only write one batch at a time, but should not be relied upon.